CalcualtorLogic.js

```diff
@@ -28,7 +28,7 @@ class Calculator {
     resultElement = null;
     constructor(selector) {
         this.hostElem = document.querySelector(selector);
-        // this.resultDiv = document.;
+        this.resultDiv = document.createElement('div');
```

Comment:
why you're doing this?

Reply:
This 
    // this.resultDiv = document.;

is not complete i did not wanted to send it
before pushing I wrote another comment
but it did not delete from git command  

These two options came in my mind for creating the property 
  this.hostElem = document.querySelector(selector);
        this.resultDiv = document.createElement('div');
without knowing what i am doing 
i am thinking may be i will know in the future when my mistakes will be corrected and this picture will complete